mmv1/products/workstations/api.yaml

```diff
@@ -447,3 +447,108 @@ objects:
               description: |
                 A list of messages that carry the error details.
               item_type: Api::Type::KeyValuePairs
+  - !ruby/object:Api::Resource
+    name: "Workstation"
+    self_link: "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}"
+    base_url: "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations"
+    create_url: "projects/{{project}}/locations/{{location}}/workstationClusters?workstationClusterId={{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}"
```

Comment:
Hmm, actually, I think this URL might be off? In the API docs (https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs.workstations/create) I see: `POST https://workstations.googleapis.com/v1beta/{parent=projects/*/locations/*/workstationClusters/*/workstationConfigs/*}/workstations`

That implies this value should be:

`projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations?workstationId={{workstation_id}}`

The API returning `500` here is wrong, but we'd just expect to get a different error from it.

Reply:
Fixing this now 👍🏼

Reply:
Fixed